src/Thenewboston.Tests/Bank/Api/BankTransactionTests.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Moq;
+using Newtonsoft.Json;
+using Thenewboston.Bank.Api;
+using Thenewboston.Bank.Models;
+using Thenewboston.Common.Http;
+using Xunit;
+
+namespace Thenewboston.Tests.Bank.Api
+{
+    
+    public class BankTransactionTests
```

Comment:
I missed this for the other tests, but please create a class with the same name as the method you are testing, and place the tests in there. Like: 

`public class GetAllTransactionsAsync  
{   
//Your tests   
}    
`

Reply:
Done